test/electron/features/native-crash.feature

```diff
@@ -2,26 +2,29 @@ Feature: Native Errors
 
   Scenario: A minidump is uploaded on native error
     When I launch an app
+    And I wait for 1 session upload
     And I click "main-process-crash"
     Then I launch an app
+    And I wait for 2 session uploads
```

Comment:
Is this wait necessary? "the total requests received by the server matches" will wait for 3 seconds already